packages/demo/scss/index.scss

```diff
@@ -61,6 +61,26 @@
     white-space: normal;
 }
 
+.docs-Placeholder {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    padding: 48px 24px;
+    text-align: center;
+    gap: 32px;
```

Comment:
same as padding, should be this in 'rem'? 🤷

Reply:
updated